src/main/java/de/bonndan/nivio/input/dto/ItemDescription.java

```diff
@@ -397,15 +230,59 @@ public String toString() {
         return FullyQualifiedIdentifier.build(environment, group, identifier).toString();
     }
 
+    /**
+     * Legacy setter for {@link StatusValue}.
+     *
+     * @param statuses
+     */
+    @Deprecated
+    public void setStatuses(List<LinkedHashMap<String, String>> statuses) {
+        statuses.forEach(map -> {
+            String key = map.get("label");
+            if (key != null) {
+                String value = map.get("status");
+                String message = map.get("message");
+                setLabel(Label.PREFIX_STATUS + Label.DELIMITER + key + Label.DELIMITER + "status", value);
+                setLabel(Label.PREFIX_STATUS + Label.DELIMITER + key + Label.DELIMITER + "message", message);
+            }
+        });
+    }
+
     @Override
     @JsonAnyGetter
     public String getLabel(String key) {
         return labels.get(key);
     }
 
     @Override
-    @JsonAnySetter
     public void setLabel(String key, String value) {
-        labels.putIfAbsent(key, value);
+        labels.put(key, value);
+    }
+
+    @JsonAnySetter
+    public void setLabel(String key, Object value) {
+        if (value instanceof String) {
+            labels.put(key.toLowerCase(), (String) value);
+            return;
+        }
+        if (value instanceof String[]) {
+            Arrays.stream(((String[]) value)).forEach(s -> setPrefixed(key, s));
+            return;
+        }
+
+        if (value instanceof List) {
```

Comment:
Is it possible to avoid the checks and casts by overloading the method?

Reply:
This JsonAnySetter is a last resort anways. See also https://stackoverflow.com/questions/22501844/multiple-any-setters-using-jackson

But yes, that would be better